originals/c.yaml

```diff
@@ -85,6 +85,11 @@
     genre: [RTS]
     theme: [Sci-Fi, Modern Military]
 
+- name: Command & Conquer: Generals
```

Comment:
I think yaml doesn't like the colon here, need to wrap in single quotes like here: https://github.com/Mailaender/osgameclones/blob/a3d6467770113543a8e2e6e8625d12d09f7612b5/originals/c.yaml#L129